prow/README.md

```diff
@@ -93,19 +93,31 @@ Prow was set up by following these instructions: https://github.com/kubernetes/t
    install your preferred one.
 
    ```shell
-   helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
-   helm install ingress-nginx/ingress-nginx --set controller.hostNetwork=true,controller.service.type="",controller.kind=DaemonSet --generate-name
-   helm repo update
+   $ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
```

Comment:
Adding `$` IMO makes the copying a bit annoying since you have to get rid of it ...

Reply:
do you want me to remove it from the $ sign from the whole document or just for helm part?

Reply:
I will be happy if we can remove it as a whole since then whole point of the document is to make it  easy to do it again

Reply:
Agreed, fixed.